docs/opentelemetry.ext.opentracing_shim.rst

```diff
@@ -0,0 +1,8 @@
+opentelemetry.ext.opentracing_shim package
+==========================================
+
+Module contents
+---------------
+
+.. automodule:: opentelemetry.ext.opentracing_shim
+   :no-show-inheritance:
```

Comment:
Why that option?

Reply:
See rationale in the commit message of d9371172e979efe72410a6d3f063f796455c00b4. If you have an idea how to address this properly, I'd love to hear :-)